experimental/PropertyDDS/packages/property-query/test/unit/multiple_queries_execution.spec.js

```diff
@@ -54,7 +54,9 @@ describe("Multiple query execution", () => {
 					],
 				}),
 			)
-				.to.be.eventually.rejectedWith('queryLanguage" must be one of')
+				.to.be.eventually.rejectedWith(
+					'Invalid query 0,queryLanguage: "[0].queryLanguage" must be [queryV1]',
```

Comment:
could you please elaborate why the rejection message has been changed?

Reply:
The rejection message changed after the major update for joi. I don't know why exactly beyond that.

Reply:
Pinging for an update @ashishswami22